packages/domain-lib/src/types/infrastructure.ts

```diff
@@ -95,6 +98,8 @@ export interface ISettlementBatchRepo {
 	// there can be multiple batches with the same name (excludes sequence number)
 	getBatchesByName(batchName: string): Promise<ISettlementBatch[]>;
```

Comment:
Same as above, no need for specific pagination method, add the optional pagination parameter to the existing method